lib/util/constants.dart

```diff
@@ -0,0 +1,4 @@
+class Constants {
```

Comment:
I prefer a specific class for 'Notification' constants, so perhaps in the provider directly